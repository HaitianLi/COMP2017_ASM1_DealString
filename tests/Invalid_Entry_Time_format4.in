Wirragulla::Cardiff::23:58:63
Wirragulla::Cardiff::30:01:23
Wirragulla::Cardiff::10:66:23
Wirragulla::Cardiff::10:59:23
Wirragulla::Cardiff::11:01:23
