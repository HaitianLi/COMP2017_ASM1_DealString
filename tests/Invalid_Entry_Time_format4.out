The next train to Cardiff from Wirragulla departs at 10:59:23
